types/voilab-pdf-table/index.d.ts

```diff
@@ -0,0 +1,224 @@
+// Type definitions for voilab-pdf-table 0.5
+// Project: https://github.com/voilab/voilab-pdf-table
+// Definitions by: Daniel Sousa <https://github.com/danielb7390>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="pdfkit" />
```

Comment:
Instead of a `/// <reference` can you get away with a simple import?

Reply:
Sure